tests/integration_tests/functional/test_drive_vhost_user.py

```diff
@@ -381,3 +384,83 @@ def test_config_change(microvm_factory, guest_kernel, rootfs):
         # Check that we can create a filesystem and mount it
         ret, out, err = vm.ssh.run(mkfs_mount_cmd)
         assert ret == 0, f"{new_size=}, {ret=}, {out=}, {err=}"
+
+
+@pytest.mark.parametrize("vcpu_count", [1, 2], ids=["1vcpu", "2vcpu"])
+def test_vhost_user_block_metrics(
```

Comment:
Should the test be in performance instead of functional? Otherwise it will run in parallel with a lot of other stuff and the metrics may be noisy.

Reply:
moved the test to performance and added more details in the commit message. Thanks !